CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ if(INSTALL_GOOGLETEST AND BUILD_TESTS)
     include(${CMAKE_ROOT}/Modules/ExternalProject.cmake)
     ExternalProject_Add(googletest
     GIT_REPOSITORY https://github.com/google/googletest
-    GIT_TAG master
+    GIT_TAG main
     SOURCE_DIR "${CMAKE_CURRENT_BINARY_DIR}/googletest-src"
     BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/googletest-build"
     TEST_COMMAND "")
```

Comment:
We should remove this option to install Google test in future via CMakeLists.txt. The reason is developers have their own environments (and hence own way of installing dependencies into it). Installing via CMake can make mess up that setting. For now this is now but in future I think we should remove it.